web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -244,17 +244,21 @@ export class LayerDialogComponent implements OnDestroy {
       );
     });
     const formId = this.dataStoreService.generateId();
+    let forms = this.layer?.forms;
```

Comment:
It's not obvious what this is doing - could you refactor this to make it more self-explanatory? (e.g. using carefully named local variables and/or functions)? Thanks!